### PR TITLE
Chart component

### PR DESCRIPTION
**Pull request by micleo2**
@sabelle

### CODE REVIEW

**Comment by sabelle** [comment]
Screen Shot 2020-02-16 at 3 36 28 PM

File: App/Components/Chart.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        labelFormatter: moneyFormat
+      },
+      xAxis: {
+        visible: true
+      },
+      dataPoint: {
+        visible: true,
+        color: "#5822a3",
+        radius: 4,
+        label: { visible: false, marginBottom: 25 }
```
Comment (uawasth2) [inline App/Components/Chart.js]: Can you also put this into your ChartStyle.js

File: App/Containers/ChartScreen/ChartScreen.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  return days.reverse();
+}
+
+function formatDate(date){
+  var dd = date.getDate();
+  var mm = date.getMonth() + 1;
+  if(dd<10) { dd='0'+dd }
+  if(mm<10) { mm='0'+mm }
+  return mm+'/'+dd;
+}
```
Comment (uawasth2) [inline App/Containers/ChartScreen/ChartScreen.js]: Change to weeks or months.

**Comment by micleo2** [comment]
:clap: :twisted_rightwards_arrows:

**Pull request merged**